src/cloud-api-adaptor/test/e2e/common_suite.go

```diff
@@ -575,9 +575,9 @@ func DoTestSealedSecret(t *testing.T, e env.Environment, assert CloudAssert, kbs
 
 // DoTestKbsKeyRelease and DoTestKbsKeyReleaseForFailure should be run in a single test case if you're chaining opa in kbs
 // as test cases might be run in parallel
-func DoTestKbsKeyRelease(t *testing.T, e env.Environment, assert CloudAssert, kbsEndpoint, resourcePath, expectedSecret string) {
-	t.Log("Do test kbs key release")
-	pod := NewBusyboxPodWithNameWithInitdata(E2eNamespace, "kbs-key-release", kbsEndpoint).GetPodOrFatal(t)
+func DoTestHTTPSKbsKeyRelease(t *testing.T, e env.Environment, assert CloudAssert, kbsEndpoint, resourcePath, expectedSecret string) {
```

Comment:
I think this can also be kept as `DoTestKbsKeyRelease` as otherwise you will need to change the `azure_test.go` to match.